frontend/src/components/pages/Dashboard/components/DeleteScheduleModal.tsx

```diff
@@ -37,18 +37,17 @@ const DeleteScheduleModal = ({
         <ModalContent py="52px" px="48px">
           <ModalHeader>
             <Text textStyle={isDesktop ? "desktopHeader3" : "mobileBodyBold"}>
-              Cancel Donation
+              Cancel One-Time Donation
             </Text>
           </ModalHeader>
           <ModalCloseButton />
           <ModalBody textStyle="mobileBody" color="hubbard.100">
             Are you sure you want to cancel your donation? This will remove all
-            linked occurances and notify all respective parties, including CFKW.
+            linked occurances and notify the donor.
```

Comment:
wrong spelling for `occurrences`. Also if this is for one time donation, what linked occurrences will be deleted anyways?

Reply:
I copied and pasted from the Figma for both of these but I'll reach out for clarification. Thanks for the comments!

Reply:
i spoke with hanlin and i think for now ill keep the wording (we can always easily change it later) but i changed the spelling here